client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/network/MessageDecoderExt.java

```diff
@@ -89,6 +89,10 @@ public static Message decode(Message.Type type, ByteBuf in, boolean decodeBody)
       case HEARTBEAT:
         return new Heartbeat();
 
+      case BUFFER_STREAM_END:
```

Comment:
be careful, add new message will cause the compatible problem for the old clients.

Reply:
worker side will check whether partition split.